backend/src/controllers/groups-controller.ts

```diff
@@ -0,0 +1,86 @@
+import { Request, Response } from 'express';
+import { v4 as uuid } from 'uuid';
+
+interface Group {
+    name:string;
+    _id:string;
+}
+
+let groups: Group[] = []; //Array of groups
+
+//Creates a group with the desired name and gives a unique ID
+export const createGroup = async (req: Request, res: Response) => {
+    const { name }: { name: string } = req.body;
```

Comment:
At the top, use `validateResult` from `express-validator` to check for errors. See the `auth-controller` or `dummy-controller` for examples.